DiceKeys/Views/Scanning/FacesReadOverlay.swift

```diff
@@ -175,15 +176,15 @@ struct RotatedTextTest: View {
     }
 }
 
-struct FacesReadOverlay_Previews: PreviewProvider {
-    static var previews: some View {
-        RotatedTextTest()
-            .previewLayout(PreviewLayout.fixed(width: 700, height: 700))
-        FacesReadOverlay(
-            renderedSize: CGSize(width: 600, height: 600),
-            imageFrameSize: CGSize(width: 600, height: 600),
-            facesRead: FaceRead.fromJson(facesReadJson)!
-        )
-        .previewLayout(PreviewLayout.fixed(width: 600, height: 600))
-    }
-}
+//struct FacesReadOverlay_Previews: PreviewProvider {
```

Comment:
Can this be fixed to create a worthwhile preview or should it just be deleted?

Reply:
Preview Fixed.